src/components/animated/_liquid.less

```diff
@@ -1,3 +1,44 @@
+
+.liquidAnimation(){
```

Comment:
Why did you put all those rules in the `liquidAnimation` mixin? It should just include the same rule as the hover that was before.